docs/contributing-guide/environment-setup/docker.md

```diff
@@ -8,25 +8,25 @@ Before proceeding, make sure you have the latest version of `docker` and `docker
 
 As of now[at the time of writing this doc], we recommend
 
-    ```bash
-    $ docker --version
-    Docker version 19.03.3, build a872fc2f86
-    $ docker-compose --version
-    docker-compose version 1.25.3, build d4d1b42b
-    ```
+```bash
+docker --version
+Docker version 19.03.3, build a872fc2f86
+docker-compose --version
+docker-compose version 1.25.3, build d4d1b42b
+```
```

Comment:
```suggestion
```bash
$ docker --version
Docker version 19.03.3, build a872fc2f86
$ docker-compose --version
docker-compose version 1.25.3, build d4d1b42b
```
```

Reply:
Please keep the prompt `$` for this block as it includes outputs.

Reply:
@vishnu-narayanan review changes updated